doc/install.rst

```diff
@@ -58,61 +59,96 @@ PyGMT requires the following libraries:
 
 The following are optional (but recommended) dependencies:
 
-* `IPython <https://ipython.org/>`__: For embedding the figures in Jupyter notebooks.
+* `IPython <https://ipython.org/>`__: For embedding the figures in Jupyter
+notebooks.
 
 
 Installing GMT and other dependencies
 -------------------------------------
 
-Before installing PyGMT, we must install GMT itself along with the other dependencies.
-The easiest way to do this is using the ``conda`` package manager.
+Before installing PyGMT, we must install GMT itself along with the other
+dependencies. The easiest way to do this is via the ``conda`` package manager.
 We recommend working in an isolated
 `conda environment <https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html>`__
-to avoid issues with competing versions of its dependencies.
+to avoid issues with conflicting versions of dependencies.
 
 First, we must configure conda to get packages from the
 `conda-forge channel <https://conda-forge.org/>`__::
 
     conda config --prepend channels conda-forge
 
-Now we can create a new conda environment with Python and all our dependencies installed
-(we'll call it ``pygmt`` but you can change it to whatever you want)::
+Now we can create a new conda environment with Python and all our dependencies
+installed (we'll call it ``pygmt`` but feel free to change it to whatever you
+want)::
 
      conda create --name pygmt python=3.9 pip numpy pandas xarray netcdf4 packaging gmt
 
-Activate the environment by running::
+Activate the environment by running the following (**do not forget this step!**))::
 
     conda activate pygmt
 
-From now on, all commands will take place inside the conda virtual environment and won't
-affect your default installation.
+From now on, all commands will take place inside the conda virtual environment
+called 'pygmt' and won't affect your default 'base' installation.
 
 
 Installing PyGMT
 ----------------
 
-Now that you have GMT installed and your conda environment activated, you can
-use ``conda`` to install the latest release of PyGMT from `conda-forge <https://anaconda.org/conda-forge/pygmt>`__::
+Now that you have GMT installed and your conda virtual environment activated,
+you can install PyGMT using any of the following methods:
+
+Using conda (recommended)
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This installs the latest stable release of PyGMT from
+`conda-forge <https://anaconda.org/conda-forge/pygmt>`__::
 
     conda install pygmt
```

Comment:
I'm thinking whether to have a quickstart installation instruction on the top since the instructions here on the page are very long and confusing (why should new users care about installing GMT first before PyGMT?). Surely we can just do:

```
conda create --name pygmt --channel conda-forge pygmt
```

Reply:
Sounds great!

Reply:
Ok, done at a243a13e35da0780a86ac16e112f9d089b72cd96.